specification/machinelearningcompute/resource-manager/Microsoft.MachineLearningCompute/2017-08-01-preview/machineLearningCompute.json

```diff
@@ -486,6 +489,13 @@
       "type": "string",
       "description": "The version of the Microsoft.MachineLearningCompute resource provider API to use.",
       "required": true
+    },
```

Comment:
@shutchings I don't entirely follow the usecase/intent of this parameter. Can you please share some more details with an example?

Reply:
When creating an operationalization cluster the resource is placed in the user's resource group and the supporting resources (ACS, app insights, ACR, storage account) are placed in a separate resource group. Right now when the user asks to delete the operationalization cluster only ACS and the operationalization cluster are deleted; ACR, storage accounts, and other resources are not deleted. This is so they still have access to their docker images, logs, telemetry, etc. But some users don't want any resources left behind when doing the delete since they no longer need any of it. With deleteAll=true all resources associated with the operationalization cluster are deleted.

Reply:
@ravbhatnagar

Reply:
@ravbhatnagar @anuchandy can I get some feedback on this? I'd like to start working on the .NET SDK/PowerShell changes.